sysstat-post-process

```diff
@@ -504,7 +504,52 @@ for my $log_file (sort readdir($dh)) {
             exit;
             }
         }
-    }
+    } elsif ($log_file =~ /^iostat-stdout.txt$/) {
+	my $log_fh = $log_file;
+        open(TMP,"$log_fh") || die "Could not open $log_fh\n";
+        my $separator = $/;
+        $/ = '';
```

Comment:
What are these lines for?

Reply:
$/ is the input record separator in perl.  It can operate in a few different modes, newline, whole-file, or, as I have it set here, it reads in stanzas that are separated by a blank line, which works really well with things that write records like iostat.

Reply:
ack